app/src/test/java/org/mozilla/fenix/ext/ConnectivityManagerTest.kt

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ext
+
+import android.net.ConnectivityManager
+import android.net.Network
+import android.net.NetworkCapabilities
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mozilla.fenix.TestApplication
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+class ConnectivityManagerTest {
+
+    private lateinit var connectivityManager: ConnectivityManager
+
+    @Before
+    fun setup() {
+        connectivityManager = mockk(relaxed = true)
+
+        mockkStatic("org.mozilla.fenix.ext.ConnectivityManagerKt")
+    }
+
+    @Test
+    fun `connectManager is online works`() {
+
+        val network = mockk<Network>()
+        val networkCapabilities = mockk<NetworkCapabilities>()
+
+        every { connectivityManager.getNetworkCapabilities(network) } returns networkCapabilities
+        every { networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_INTERNET) } returns true
+        every { networkCapabilities.hasCapability(NetworkCapabilities.NET_CAPABILITY_VALIDATED) } returns true
+
+        Assert.assertEquals(true, connectivityManager.isOnline(network))
```

Comment:
not: `assertTrue/False(...)` without `Assert.`

Reply:
Yes, i know the drill now. :)

Reply:
Still needs to use `assertTrue/False(...)` instead of `assertEquals(true/false, ...)` :)

Reply:
My bad. Impatience level 9000. Fixed.